src/components/SettingsDropdown.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import Gear from '../assets/settings.png';
+import Gear from '../assets/new-settingsIcon.png';
```

Comment:
small fix but I recommend using svg's for icons, which yields higher quality images (it is currently quite pixelated)
you can go to the navbar frame on Figma and download the settings icon as an svg: https://www.figma.com/file/vWdNmoKEuSF24x9OYN9vJh/KS-NSEVP-Wireframes?node-id=426%3A11340